source/includes/debt-settlement.md.erb

```diff
@@ -719,7 +723,9 @@ X-GOJI-CLIENT-ID: 79f33f3c-86e0-4613-ba49-9fac3c6f0eac
       },
       "type": "INTEREST",
       "status": "PENDING",
-      "investmentId": "5ab5df55-8e5e-4380-a94c-6fcac3a9a6ac"
+      "investmentId": "5ab5df55-8e5e-4380-a94c-6fcac3a9a6ac",
+      "clientId": "2cc3df55-8e1a-4220-a95c-2bcaa1a9a6ac",
```

Comment:
Can you double-check the clientId being returned is the clientId and not investorId (settlement side?) And update the docs accordingly

Reply:
I did, it is indeed the clientId.